tests/security/security_tests.lst

```diff
@@ -38,4 +38,4 @@ tests/security/attributes/run_test.pl --scenario FullMsgEncrypt_SubMsgSign_Paylo
 tests/security/attributes/run_test.pl --scenario FullMsgEncrypt_SubMsgEncrypt_PayloadEncrypt
 #tests/security/attributes/run_test.pl -scenario FullMsgSign_PayloadEncrypt_Frag
 tests/security/TopicWithoutPermissions/run_test.pl
-
+tests/DCPS/ParticipantLocationTopic/run_test.pl
```

Comment:
Based on the other lst file, this should have (at least) CXX11 !NO_BUILT_IN_TOPICS.  Since this is the security_tests file we already know it isn't safety profile and it is RTPS,